docs/taskruns.md

```diff
@@ -307,6 +308,38 @@ If multiple `steps` are defined in the `Task` invoked by the `TaskRun`, we will
 `spec.steps` of the `Task`, when the `TaskRun` is accessed by the `get` command, e.g.
 `kubectl get taskrun <name> -o yaml`. Replace \<name\> with the name of the `TaskRun`.
 
+## Status
+
+As a TaskRun completes, it's `status` field is filled in with relevant information for
+the overall run, as well as each step.
+
+The following example shows a completed TaskRun and it's `status` field:
+
+```yaml
+completionTime: "2019-08-12T18:22:57Z"
+conditions:
+- lastTransitionTime: "2019-08-12T18:22:57Z"
+  message: All Steps have completed executing
+  reason: Succeeded
+  status: "True"
+  type: Succeeded
+podName: status-taskrun-pod-6488ef
+startTime: "2019-08-12T18:22:51Z"
+steps:
+- container: step-hello
+  imageID: docker-pullable://busybox@sha256:895ab622e92e18d6b461d671081757af7dbaa3b00e3e28e12505af7817f73649
+  name: hello
+  terminated:
+    containerID: docker://d5a54f5bbb8e7a6fd3bc7761b78410403244cf4c9c5822087fb0209bf59e3621
+    exitCode: 0
+    finishedAt: "2019-08-12T18:22:56Z"
+    reason: Completed
+    startedAt: "2019-08-12T18:22:54Z"
+  ```
+
+Fields include start and stop times for the `TaskRun` and each `Step`, and exit codes.
+For each step we also include the fully-qualified image used, with the digest.
```

Comment:
u da best!

Reply:
if you need to revisit this PR again anyway, could you move the` ## Steps` section to be a subsection of `status`? (t would mean updating the TOC at the top too) ❤️